db/app.models.js

```diff
@@ -6,3 +6,17 @@ exports.selectAllTopics = () => {
   });  
 };
 
+exports.selectArticles = (id) => {
+let query = "SELECT * FROM articles"
+
+const queryValues = []
+
+if (id){
+  query += " WHERE article_id = $1";
+  queryValues.push(id)
+}
+  query += " ;"
```

Comment:
This if statement is not necessary

Reply:
im trying to create a model function for getting all articles and a single one

Reply:
These two model functions should not be the same and should be separated, please revisit the learn to code instructions for more detail.